kms++/src/card.cpp

```diff
@@ -216,6 +216,10 @@ void Card::setup()
 	m_has_atomic = false;
 #endif
 
+	if (getenv("KMSXX_ENABLE_CLIENT_CAP_STEREO_3D") != 0) {
+		drmSetClientCap(m_fd, DRM_CLIENT_CAP_STEREO_3D, 1);
+	}
+
```

Comment:
This is not how the other ones work. The idea with KMSXX_DISABLE_UNIVERSAL_PLANES and KMSXX_DISABLE_ATOMIC is that by default the features are enabled. They are disabled only if a) the feature is not available (drmSetClientCap fails), or b) the env variable is set, to explicitly disable the feature.

I'm not familiar with STEREO_3D. What does it mean if it's enabled? What changes? My guess is that we should just enable it by default.

Reply:
By default only mono modes are reported from kernel. The STEREO_3D capability also reports the 3d variants.
E.g. here is output from a 3D capable TV on Pi without the cap:
http://sprunge.us/NSRzeY
And here with the 3D cap:
http://sprunge.us/qP1frS
You get an extra 14 modes reported here, 
It could be enabled by default, but it creates some extra noise (assuming most users don't care about 3D).

Reply:
In that case perhaps it's better to enable 3d by default in kms++, but in kmsprint only show the 3d modes if a cmdline parameter ("-3d"?) is given.

That said, at least in your example, it doesn't look that bad. So I think we might as well have 3d always enabled and kmsprint shows them all. If at some point the extra noise feels too much, we can add the -3d parameter to kmsprint.